Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/impl/workbench/PublicPageFactory.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.spring.quadriga.domain.factory.impl.workbench;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.factory.workbench.IPublicPageFactory;
+import edu.asu.spring.quadriga.domain.impl.workbench.PublicPage;
+import edu.asu.spring.quadriga.domain.impl.workbench.PublicStatisticsPage;
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+/**
+ * Factory class to create Public Page object
+ * 
+ * @author Prasanth Priya Nesan
+ * 
+ */
+@Service
+public class PublicPageFactory implements IPublicPageFactory {

```

Comment:
This class was taken from story 86, as I have to refer previous story to comply with it.